components/calendar/src/calendar_arithmetic.rs

```diff
@@ -302,6 +339,30 @@ pub fn ordinal_solar_month_from_code(code: types::MonthCode) -> Option<u8> {
     None
 }
 
+/// For lunar calendars, get the month number from the month code
+///
+/// Similar to [`ordinal_solar_month_from_code`], but adds 1 if the month code
+/// is four characters long (indicating 'L' for leap or 'I' for increment were appended)
+pub fn ordinal_lunar_month_from_code(code: types::MonthCode) -> Option<u8> {
```

Comment:
issue: this is not a correct implementation, see below. the impl of new_from_lunar_codes is going to have to be calednar-specific

Reply:
Can we add this function to the `CalendarArithmetic` trait?